.github/CODEOWNERS

```
@@ -6369,7 +6369,7 @@
 /types/selectables/                                     @renjfk
 /types/selectize/                                       @adidahiya @naBausch
 /types/selectn/                                         @bendaly818
-/types/selenium-standalone/                             @mgrybyk
+/types/selenium-standalone/                             @SanderDeWaal1992
```

```suggestion
/types/selenium-standalone/                             @mgrybyk
```
so actually, reverts this line as outcome

---

It was already changed, refresh the page